### PR TITLE
Update to Polkadot-SDK 1.6.0

### PR DESCRIPTION
**Pull request by rphmeier**

### CODE REVIEW

**Comment by rphmeier** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #203** <a href="https://app.graphite.dev/github/pr/thrumdev/blobs/203?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈
    * **PR #201** <a href="https://app.graphite.dev/github/pr/thrumdev/blobs/201?utm_source=stack-comment-icon" target="_blank">Graphite</a> 

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/thrumdev/blobs/203?utm_source=stack-comment).

File: sugondat/chain/node/src/command.rs [hunk 1]
```diff
@@ -177,12 +177,12 @@ pub fn run() -> Result<()> {
                 Ok(())
             })
         },
-		Some(Subcommand::ExportGenesisState(cmd)) => {
+		Some(Subcommand::ExportGenesisHead(cmd)) => {
```
Comment (pepyakin) [inline sugondat/chain/node/src/command.rs:180]: I wonder if this breaks zombienet...

File: sugondat/chain/node/src/cli.rs [hunk 2]
```diff
@@ -34,7 +34,7 @@ pub enum Subcommand {
     ExportGenesisMetadata(export_genesis_metadata::ExportGenesisMetadataCmd),
 
     /// Export the genesis state of the parachain.
-    ExportGenesisState(cumulus_client_cli::ExportGenesisStateCommand),
+    ExportGenesisHead(cumulus_client_cli::ExportGenesisHeadCommand),
```
Comment (pepyakin) [inline sugondat/chain/node/src/cli.rs:37]: I know that the documentation above is not really informative, but perhaps should be updated for consistency.

**Review by pepyakin — APPROVED** [review]

**Comment by pepyakin** [comment]
please kindly rebase.

**Pull request merged**